lib/mediawiki_api/wikidata/wikidata_client.rb

```diff
@@ -47,6 +47,18 @@ def create_claim(entity_id, snaktype = "value", property_id, value_data)
         action(:wbcreateclaim, token_type: "csrf", entity: entity_id, snaktype: snaktype, property: property_id, value: value_data)
       end
 
+      def get_entities(ids, site_id)
+        action(:wbgetentities, token_type: false, ids: ids, sites: [site_id])
```

Comment:
The `sites` parameter only makes sense together with `titles`. You won't be able to use the API like this. Either provide one or more `ids`, or provide one or more pairs of `sites` and `titles`.

Reply:
Oops, I was mistaken in my understanding.  I'm confusing `sites` and `languages`. Thanks for your comment!

Reply:
I removed the parameter `site_id` from `get_entities`, and add a new method `get_entities_by_titles` to get items with `titles` and `sites`.